apps/netlify/frontend/src/app/edit-site-modal.js

```diff
@@ -218,9 +267,13 @@ export const EditSiteModal = ({
                   required
                 >
                   {isNewSite && (
-                    <Option value={PICK_OPTION_VALUE}>Pick site</Option>
+                    <Option value={PICK_OPTION_VALUE}>
+                      {availableNetlifySites.length === 0
+                        ? 'No Netlify sites available'
+                        : 'Select a Netlify site'}
+                    </Option>
                   )}
```

Comment:
This still feels strange to me. Can we remove that option when the account has one site? 
Imo, it's an odd UX to allow the user to select an invalid/placeholder site

Reply:
This isn't actually selectable though. It just means that we render an informative placeholder. 

Maybe we shouldn't let the user open the modal at all if there are no sites?

Reply:
I think we already prevent the user from opening the modal when all sites are configured